python/chilldkg_ref/vss.py

```diff
@@ -74,6 +74,32 @@ def commitment_to_secret(self) -> GE:
     def commitment_to_nonconst_terms(self) -> List[GE]:
         return self.ges[1 : self.t()]
 
+    def invalid_taproot_commit(self) -> Tuple[VSSCommitment, Scalar]:
+        # Create VSS commitment that Taproot-commits to a message that is
+        # invalid for BIP 341 script path spends.
+        #
+        # Specifically, for a VSS commitment `com`, we have:
+        # `com.add_invalid_taproot_commitment().commitment_to_secret() = com.commitment_to_secret() + t*G`.
+        #
+        # The tweak `t` commits to a message chosen such that the new
+        # `commitment_to_secret()` is unspendable via a BIP 341 Taproot script
```

Comment:
consistency nit: (multiple times)
```suggestion
        # `commitment_to_secret()` is unspendable via a BIP341 Taproot script
```

Reply:
Should we instead add a space between "BIP" and the number everywhere? This is how the old and new BIP 2 refers to BIPs.

Reply:
Sure, happy with either way.